lib/keka/result.rb

```diff
@@ -11,7 +11,27 @@ def ok?
       is_success
     end
 
+    def msg
+      return @msg if @msg.is_a?(String)
+      return @msg.full_messages.join(', ') if active_model_error?
+
+      @msg
+    end
+
+    def errors
+      return {} if ok?
+      return {base: [@msg]} if @msg.is_a?(String)
+      return @msg.messages if active_model_error?
```

Comment:
let's rename this instance variable to something like `errors` or `msg_or_errors`. `msg` implies a string data type.